source/content/php-versions.md

```diff
@@ -26,12 +26,14 @@ The recommended PHP versions available on Pantheon are:
 
 - [7.3](https://v73-php-info.pantheonsite.io/)
 - [7.2](https://v72-php-info.pantheonsite.io/)
-- [7.1](https://v71-php-info.pantheonsite.io/)
 
 Click on the links above to see the complete PHP info for each version, including the list of supported PHP extensions.
 
 ### EOL PHP Versions
-Pantheon also makes PHP [7.0](https://v70-php-info.pantheonsite.io/), [5.6](https://v56-php-info.pantheonsite.io/), [5.5](https://v55-php-info.pantheonsite.io/), and [5.3](https://v53-php-info.pantheonsite.io/) available on the platform, although these are end-of-life (**EOL**), and should not be used unless absolutely necessary.
+Pantheon also makes PHP [7.1](https://v71-php-info.pantheonsite.io/), PHP [7.0](https://v70-php-info.pantheonsite.io/), [5.6](https://v56-php-info.pantheonsite.io/), and [5.5](https://v55-php-info.pantheonsite.io/) available on the platform, although these are end-of-life (**EOL**), and should not be used unless absolutely necessary. 
+
+### PHP Versions Being Retired
+PHP [5.3](https://v53-php-info.pantheonsite.io/) is being retired from the platform. Sites using PHP 5.3 will continue to serve pages, but new development cannot be done. To resume development on a site using a retired version of PHP, first upgrade the PHP version on the development site.
```

Comment:
To follow up with my comment in the main PR comment thread, my concern is this section being time-specific. I would suggest announcing the retirement of PHP 5.3 via the Changelog, blog, and any other content format that's designed to be consumed when published. Then this section can be removed, and the rest is an automatic +1.

Reply:
Actually, what I think is needed here is just a better description of the status.

I intended "being retired" to be a new state for software to be in, like the "EOL" state.

A handy comparison table:

State               | Upgrade ASAP | Site still works | Development still works
------------- | --------------- | --------------- | ------------------------
EOL                 | Yes                   | Yes                    | Yes
Being retired   | Yes                  | Yes                     | *No*

I intended "being retired" to be a perpetual state with no particular end date, i.e. we might never remove this version from the platform, we just won't let you do development with it any longer.

I would prefer it if the documentation described this new "less supported" state for a while (indeterminate duration, until Product decides to do something else); in that respect, I would prefer to change the adjective from "being retired" to something else rather than remove 5.3 support with nothing more than changelog documentation, if there is a better way to describe it.

Reply:
Thanks for the additional context. That makes a heaping spoonful of sense. I also think that, like in your explanation above, a table is the optimal way to display what versions are in what state, with a definition list or popovers to define the various states.